### DISCUSSION TITLE
在使用pnpm install 安装命令后，出现postinstall阶段的turbo run stub 出现错误

### DISCUSSION THREAD

**Discussion by 1shirely1** [vbenjs/vue-vben-admin#4388] [Q&A]
版本：
nodejs：v20.17.0
pnpm：9.10.0
出现的问题如下：
![1726195820352](https://github.com/user-attachments/assets/09de3af2-2167-458a-b628-193e76bb5957)
目前没有在网络上找到类似的解决方案，烦请各位看一下，希望得到解答

**Comment by peachMT** [comment]
请问有解决吗  我也出现了同样的问题

**Comment by anncwb** [comment]
这个问题我没办法复现，得看谁能在window系统环境帮忙排查下原因

**Comment by risu729** [comment]
test
